scripts/ironic-common.sh

```diff
@@ -22,7 +26,7 @@ export LISTEN_ALL_INTERFACES="${LISTEN_ALL_INTERFACES:-"true"}"
 # Wait for the interface or IP to be up, sets $IRONIC_IP
 function wait_for_interface_or_ip() {
   # If $PROVISIONING_IP is specified, then we wait for that to become available on an interface, otherwise we look at $PROVISIONING_INTERFACE for an IP
-  if [ ! -z "${PROVISIONING_IP}" ];
+  if [ ! -z "${PROVISIONING_IP:-}" ];
```

Comment:
I think ! -z can be shortened to -n

Reply:
yep, I'll change that, thanks for the comment